generator/Utils.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-underscore-dangle */
+import { fileURLToPath } from 'url';
+import path, { dirname } from 'path';
+
+export const __filename = fileURLToPath(import.meta.url);
+export const __dirname = dirname(__filename);
+
+// Paths that will be used
+export const typeFile = path.resolve(__dirname, '../../types/index.d.ts');
+export const mainFile = path.resolve(__dirname, '../../src/index.ts');
+export const apiMapFile = path.join(__dirname, 'apiMap.json');
```

Comment:
I think this file is git ignored. When I run `npm run generate:main` it complains that it doesn't exist.

Reply:
The API Map is generated when you run the `generate:types` to always be updated with the lastest schema available on the directory